bot/extensions/polls/commands.py

```diff
@@ -0,0 +1,97 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+
+from bot import core
+from bot.extensions.polls.views import CreatePollView
+from utils.transformers import MessageTransformer
+
+
+class Polls(commands.GroupCog, group_name="poll"):
+    def __init__(self, bot: core.DiscordBot):
+        self.bot = bot
+
+    @property
+    def reactions(self):
+        return {
```

Comment:
Why not just `self.reactions = {...}`
This is also defined 3 different times I think we can put it in another file mhm

Other than this pety comment the PR looks good now